daemon/plugin/plugin.go

```diff
@@ -0,0 +1,241 @@
+package plugin // import "github.com/docker/docker/daemon/plugin"
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/containerd/containerd/errdefs"
+	"github.com/docker/docker/daemon"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/pkg/errors"
+)
+
+const (
+	// GRPCPlugin implements a grpc service
+	GRPCPlugin Type = "io.docker.grpc.v1"
```

Comment:
I'm thinking if we should use `com.docker...` instead of `io.docker...`. Looking through the codebase, it looks like we use `com.docker..` for swarmkit labels, libnetwork options, and some other places, so it might be more consistent.